go.mod

```diff
@@ -1,19 +1,126 @@
 module kubevirt.io/containerdisks
 
-go 1.16
+go 1.17
+
+replace (
+	github.com/openshift/api => github.com/openshift/api v0.0.0-20211217221424-8779abfbd571
+	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20211209144617-7385dd6338e3
+
+	k8s.io/client-go => k8s.io/client-go v0.23.5
+)
 
 require (
-	github.com/containers/image/v5 v5.17.0
-	github.com/docker/distribution v2.7.1+incompatible
+	github.com/containers/image/v5 v5.20.0
+	github.com/docker/distribution v2.8.1+incompatible
 	github.com/ghodss/yaml v1.0.0
-	github.com/google/go-containerregistry v0.7.0
-	github.com/onsi/gomega v1.10.3
+	github.com/google/go-containerregistry v0.8.1-0.20220328141311-efc62d802606
+	github.com/onsi/gomega v1.19.0
+	github.com/opencontainers/image-spec v1.0.3-0.20220114050600-8b9d41f48198
 	github.com/pkg/errors v0.9.1
+	github.com/sethvargo/go-retry v0.2.3
```

Comment:
Is it ok to add a dependency on someone's personal github repo?

Reply:
Removed the dependency